cmd/options/node_options_test.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"testing"
+
+	flag "github.com/spf13/pflag"
+)
+
+func TestNodeOptions(t *testing.T) {
+	testCases := []struct {
+		name  string
+		flag  string
+		found bool
+	}{
+		{
+			name:  "lookup desired flag",
+			flag:  "volume-attach-limit",
```

Comment:
Remove this.

Reply:
good catch, removed + added ./cmd/... to test path in Makefile